### PR TITLE
Remove unused functionality

### PR DESCRIPTION
**Pull request by psbanka**
#MAJOR#

#Changelog
A lot of the client-side functionality we didn't end up using and we were wasting resources trying to maintain it. Therefore, let's pull it out and focus on the core use-case of webdriverio-server

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+# Configure nginx
+RUN cp /etc/nginx/nginx.conf /etc/nginx/nginx.conf.default
+COPY nginx.conf /etc/nginx/nginx.conf
+RUN service nginx reload
+
+
+# Install nodeenv and node version (defined by NODENV_VERSION environment variable)
+ENV NODENV_VERSION 6.9.2
+ENV ENV_DIR /.ci-env
+ENV DISPLAY :0
```
Comment (sandersky) [inline Dockerfile]:
```Dockerfile
ENV NODENV_VERSION 6.9.2 \
    ENV_DIR /.ci-env \
    DISPLAY :0
```

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+# RUN echo 'nvm install ${NODE_VERSION}' >> $HOME/.profile
+# RUN echo 'nvm alias default ${NODE_VERSION}' >> $HOME/.profile
+# RUN echo 'echo "Install node@${NODE_VERSION} finished."' >> $HOME/.profile
+RUN nenv npm install bower -g
+
+RUN nenv npm install -g webdriverio-server
+RUN nenv webdriverio-server-init
+
+ENV PORT 3001
+ENV DEBUG server
```
Comment (sandersky) [inline Dockerfile]:
```Dockerfile
ENV PORT 3001 \
    DEBUG server
```

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+RUN tar -zxf jdk-8u65-linux-x64.tar.gz -C /usr/lib/jvm
+RUN rm jdk-8u65-linux-x64.tar.gz
+
+# Configure Java
+RUN update-alternatives --install "/usr/bin/java" "java" "/usr/lib/jvm/jdk1.8.0_65/bin/java" 1
+RUN update-alternatives --install "/usr/bin/javac" "javac" "/usr/lib/jvm/jdk1.8.0_65/bin/javac" 1
+RUN update-alternatives --install "/usr/bin/javaws" "javaws" "/usr/lib/jvm/jdk1.8.0_65/bin/javaws" 1
+RUN chmod a+x /usr/bin/java
+RUN chmod a+x /usr/bin/javac
+RUN chmod a+x /usr/bin/javaws
```
Comment (sandersky) [inline Dockerfile]:
```Dockerfile
RUN chomd a+x /usr/bin/java \
    /usr/bin/javac \
    /usr/bin/javaws
```

File: npmrc [hunk 4]
```diff
@@ -0,0 +1,3 @@
+# Setup NPM auth by adding auth into into .npmrc
+email=teamcity.ciena.com
+registry=https://artifactory.ciena.com/api/npm/blueplanet-npm
```
Comment (sandersky) [inline npmrc]: This entire file looks like it should not be in a public repo.

Comment (psbanka) [inline npmrc]: true. However it is necessary for it to work. Thought I'd leave this in for now and figure out a Better Way later on. Do you have any suggestions?

File: Dockerfile (hunk 2 above)
Comment (psbanka) [inline Dockerfile]: It doesn't look like `ENV` works this way.

Comment (sandersky) [inline Dockerfile]: Derp, you need `=` in those lines

```Dockerfile
ENV PORT=3001 \
    DEBUG=server
```

**Comment by sandersky** [comment]
:+1:

**Pull request merged**